app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<h2>Sign up</h2>
```

Comment:
Have you made any changes to this file? If you haven't you don't need to add it. By navigating to the route `devise` will automatically goes to this view.

It is only necessary to add it if you want to change the form